content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+    
+This work was done as a part of [KEP #4633](https://github.com/kubernetes/enhancements/issues/4633), by [SIG Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
+
+### Per-plugin callback functions for accurate requeueing in kube-scheduler enhancements
+
+Enhance the scheduling throughput with more efficient scheduling retry decisions by per-plugin callback functions (QueueingHint). 
+We've implemented QueueingHints in all plugins, and are moving ahead enabling this feature by default.
+
+This work was done as a part of [KEP #4247](https://github.com/kubernetes/enhancements/issues/4247), by [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+### Recover from volume expansion failure
+
+This feature moved to Beta in v1.32, allowing users to recover from volume expansion failure by retrying with a smaller size. This enhancement ensures that volume expansion is more resilient and reliable, reducing the risk of data loss or corruption during the process.
+
+This work was done as a part of [KEP #1790](https://github.com/kubernetes/enhancements/issues/1790), by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Volume group snapshot
+
+This feature introduces a VolumeGroupSnapshot API to take a snapshot of multiple volumes together.
+Allows users to take a snapshot of multiple volumes together, ensuring data consistency across the volumes.
+
+This work was done as a part of [KEP #3476](https://github.com/kubernetes/enhancements/issues/3476), by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Structured parameter support
+
+The core part of Dynamic Resource Allocation (DRA), the structured parameter support, got promoted to beta.
+Thanks to this feature, the kube-scheduler and Cluster Autoscaler can handle respectively simulate claim allocation themselves without relying on a third-party driver.
+
+This work was done as a part of [KEP #4381](https://github.com/kubernetes/enhancements/issues/4381), by WG Device Management (a cross functional team containing [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node), [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling) and [SIG Autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling)).
+
+## New features in Alpha
+
+_This is a selection of key improvements introduced as alpha features in the v1.32 release._
+    
+### Asynchronous preemption in the Kubernetes Scheduler
+
+Enhance the scheduling throughput when the preemption is involved, by making some expensive operations within the preemption asynchronously.
```

Comment:
```suggestion
This feature enhances the scheduling throughput when preemption is involved by making some expensive
preemption operations asynchronous.
```

Be good if we could say what we mean by "expensive"

Reply:
Thanks @shannonxtreme for this feedback.
"Expensive operations" are not explicitly in the enhancement description and is a term to cover: time-consuming, scheduler performance

We can do two things :
1. Clarify it with an example we can dd in parenthesis.
`expensive operations (e.g., resource-intensive API calls like updating pod conditions and deleting pods, which can delay scheduling cycles)`

2. We can also change that phrase and add `Resource-intensive operations` or `Heavy operations` instead of "Expensive operations" to refer to all high resource demands, such as CPU or network calls, mentioned in the enhancement.

What do you think? 🙂

Reply:
I like resource-intensive :)

Reply:
Great, I'll use that instead!

- want to fix every of your comments in a single commit so I'm working local for now @shannonxtreme 
Thanks for all the great feedback, much appreciated